### PR TITLE
add a user and a group for KA Lite helper

### PR DESCRIPTION
**Pull request by xaiki**
Signed-off-by: Niv Sardi <xaiki@evilgiggle.com>

### CODE REVIEW

File: group.master [hunk 1]
```diff
@@ -60,3 +60,4 @@ metrics:x:120:
 systemd-journal:x:121:
 geoclue:x:122:
 app-manager:x:123:
+kalite:x:124:
```
Comment (mariospr) [inline group.master]: Don't remove the last empty line

File: passwd.master [hunk 2]
```diff
@@ -34,3 +34,4 @@ geoclue:*:114:122::/var/lib/geoclue:/usr/sbin/nologin
 speech-dispatcher:*:115:29:Speech Dispatcher,,,:/var/run/speech-dispatcher:/usr/sbin/nologin
 app-manager:*:116:123:Endless App Manager:/var/endless:/usr/sbin/nologin
 hplip:*:117:7:HPLIP system user,,,:/var/run/hplip:/usr/sbin/nologin
+kalite:*:118:8:KA Lite Helper system user,,,:/var/lib/kalite/:/bin/sh
```
Comment (mariospr) [inline passwd.master]: Group should match the newly created 'kalite' group (124).
Shell can probably be nologin, too.

**Comment by mariospr** [comment]
Besides the couple of comments I made in the PR, I think it would be nice if you could maintain the usual "structure" we use for commit messages showing up in the git history, as in:
```
    <title>
    
    <description if needed>
    
    <link to phabricator>
```

That said, the description is probably not needed for this change, but at least adding a link to https://phabricator.endlessm.com/T4489 would be nice :-). Thanks!

**Comment by xaiki** [comment]
@mariospr all fixed
i had set shell to /bin/sh when we were trying to hack getting a login shell but we have your patch in flatpak around that

**Comment by mariospr** [comment]
@xaiki Thanks. Looks good to me, but let's wait until the rest of the bits are ready to be merged before getting this one in.

**Comment by mariospr** [comment]
In the end, it seems we need the kalite user to login or this won't work, so I added that back in a commit

**Pull request merged**